spec/features/variable_drafts/forms/validation_spec.rb

```diff
@@ -343,4 +339,25 @@
     end
   end
 
+  context 'multiple simple fields' do
+    before do
+      visit edit_variable_draft_path(@draft, 'variable_characteristics')
+    end
+
+    context 'when adding one new field' do
+      before do
+        fill_in 'variable_draft_draft_characteristics_index_ranges_lat_range_0', with: '1'
+        fill_in 'variable_draft_draft_characteristics_index_ranges_lon_range_0', with: '1'
+        within '.nav-top' do
+          click_on 'Save'
+        end
+      end
+
+      it 'displays validation error messages' do
+        expect(page).to have_css('#variable_draft_draft_characteristics_index_ranges_lat_range_error', text: 'Lat Range has too few items')
+        expect(page).to have_css('#variable_draft_draft_characteristics_index_ranges_lon_range_error', text: 'Lon Range has too few items')
+      end
```

Comment:
I feel like we should add another item to one or both ranges, to test that the error goes away.

Reply:
Yes, will add that test